dll/win32/iphlpapi/iphlpapi_main.c

```diff
@@ -3200,10 +3200,10 @@ DWORD WINAPI DECLSPEC_HOTPATCH GetAdaptersAddresses(ULONG Family,ULONG Flags,PVO
 
             /* Friendly name */
             if (!(Flags & GAA_FLAG_SKIP_FRIENDLY_NAME))
-                requiredSize += strlen((char *)ifInfo.if_info.ent.if_descr) + 1; //FIXME
+                requiredSize += ifInfo.if_info.ent.if_descrlen + 1; //FIXME
```

Comment:
why still marked with FIXME comment?

Reply:
I don't know what this FIXME is for, so kept it :)

Reply:
git blame may indicate when this has been introduced and perhaps why...